dev-tools/whitelist.sh

```diff
@@ -0,0 +1,13 @@
+uhb_whitelist -f ${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVE-in-testing.txt -o ${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVE-in-testingTEMP.txt
+uhb_whitelist -f ${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVE.txt -o ${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVETEMP.txt
+uhb_whitelist -f ${TRAVIS_BUILD_DIR}/phishing-domains-INACTIVE-in-testing.txt -o ${TRAVIS_BUILD_DIR}/phishing-domains-INACTIVE-in-testingTEMP.txt
+uhb_whitelist -f ${TRAVIS_BUILD_DIR}/phishing-domains-INACTIVE.txt -o ${TRAVIS_BUILD_DIR}/phishing-domains-INACTIVETEMP.txt
+uhb_whitelist -f ${TRAVIS_BUILD_DIR}/phishing-domains-INVALID-in-testing.txt -o ${TRAVIS_BUILD_DIR}/phishing-domains-INVALID-in-testingTEMP.txt
+uhb_whitelist -f ${TRAVIS_BUILD_DIR}/phishing-domains-INVALID.txt -o ${TRAVIS_BUILD_DIR}/phishing-domains-INVALIDTEMP.txt
+grep . ${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVE-in-testingTEMP.txt >${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVE-in-testing.txt
```

Comment:
"Greping" is not great as you do a filtering which is done by PyFunceble later. 
What you do is removing even the `INVALID` one. And we do not want that here as it's the job of PyFunceble.

The difference between "greping" and PyFunceble is that PyFunceble allow us to keep a track overtime. And why not in the future have an interface which let us know more about each of the input sources which are going to come in the future.

So please no "greping".

You may think: But PyFunceble take so much time so why not filtering now ? Well PyFunceble take much time now but not anymore with the 2.0.0 version which will comes (if everything is going right) tomorrow as `PyFunceble-dev`  (so reported here as we do `pip3 install PyFunceble-dev`) and in the coming 2 weeks as `PyFunceble` (stable)

Reply:
Ok, Ill remove the greping